src/Admin/AbstractAdmin.php

```diff
@@ -1404,6 +1419,12 @@ public function getTemplates()
      */
     public function getTemplate($name)
     {
+        @trigger_error(sprintf(
+            'Method "%s" is deprecated since sonata-project/admin-bundle 3.34 and will be removed in 4.0. '
+            .'Use TemplateRegistry services instead.',
```

Comment:
```suggestion
            'Method "%s()" is deprecated since sonata-project/admin-bundle 3.34 and will be removed in 4.0.'
            .' Use TemplateRegistry services instead.',
```